src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -184,11 +182,6 @@ const transform = (
             reverseField =>
               reverseField.fieldMainLocation && reverseField.entityPublished,
           )
-          .sort((a, b) =>
-            a.fieldNicknameForThisFacility.localeCompare(
-              b.fieldNicknameForThisFacility,
-            ),
-          )
```

Comment:
Should we sort by title instead?

Reply:
I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman